charts/drax/charts/dashboard/templates/configmap-core-ui.yaml

```
@@ -6,6 +6,18 @@
 {{- define "accelleran.dashboard.configMap.coreUi.args" -}}
 {{- $ := . -}}
 
+{{- $currentProduct := $.Values.global.accelleranProduct | default "" -}}
```

Maybe we should always change to uppercase to also support `drax` etc. as a value?

```suggestion
{{- $currentProduct := $.Values.global.accelleranProduct | default "" | upper -}}
```